software/app/src/main/java/com/example/signbuddy/MainActivity.java

```diff
@@ -98,6 +126,38 @@ private void openQuizActivity(View v) {
         startActivity(intent);
     }
 
+    private void collectButtonCallback(View v) {
+        EditText collectLetter = findViewById(R.id.collectLetter);
+        gestureLetter = collectLetter.getText().toString();
+        if (gestureLetter.matches("")) {
+            Toast.makeText(this, "No letter specified!", Toast.LENGTH_SHORT).show();
+        } else {
+            collectButton.setEnabled(false);
+            collectButton.setText(R.string.collecting);
+            byte[] cmd;
+            if ("jzJZ".contains(gestureLetter)) {
```

Comment:
what is jxJZ

Reply:
oh, got it.
can you change this because it's not easy to understand

or at least add a comment

Reply:
Yeah I'll add a comment